src/core.js

```diff
@@ -298,7 +298,7 @@ var EasyAutocomplete = (function(scope) {
 											.mouseout(function() {
 												config.get("list").onMouseOutEvent();
 											})
-											.html(template.build(highlight(elementsValue, phrase), listData[j]));
+											.html(template.build(highlight(htmlEntities(elementsValue), phrase), listData[j]));
```

Comment:
It would be nice to add a config flag for this HTML injection feature, so a user can decide whether they want to use it or not. It can be enabled by default.

Reply:
Yeah you are right. I forget the flags. I will check it tonight.